pyscitt/pyscitt/client.py

```diff
@@ -78,7 +94,7 @@ class BaseClient:
 
     url: str
     auth_token: Optional[str]
-    member_auth: Optional[Tuple[str, str]]
+    member_auth: Optional[Tuple[str, str, str]]
```

Comment:
I think this would be cleaner by defining an abstract class, with two implementations (KeyVaultSignClient and a local credentials one).

For example:
```
class MemberAuthenticationMethod(ABC):
    def sign_proposal(self, data: bytes) -> bytes: ...
```

and then in BaseClient, the field would be `member_auth: MemberAuthenticationMethod`.

The `client_arguments.py` file would create the right object depending on the arguments.